KarhooUISDK/Screens/BookingScreen/Map/PickupOnlyStrategy.swift

```diff
@@ -102,11 +95,8 @@ final class PickupOnlyStrategy: PickupOnlyStrategyProtocol, BookingMapStrategy,
             }
         }
 
-        if Karhoo.configuration.authenticationMethod().isGuest() {
-            focousOnPickup()
```

Comment:
the `focousOnPickup` seems no longer needed? 

Is it possible to check if we have location permission? if we dont have location permission, perhaps we can display a dialog asking user to go to settings and enable the location permission?

Reply:
If it is not achievable to do it as part of this task, you can add a ticket to tackle it separately 

Here is an example of the dialog in Google Maps

<img width="417" alt="Screenshot 2021-05-26 at 16 37 15" src="https://user-images.githubusercontent.com/12498117/119689313-a62bba00-be40-11eb-92ec-e8765e378090.png">

Turn on your location settings

Reply:
Added back the focus on pickup. For the popup, we should have a different ticket